content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+---
+title: Asset Tracking
+section: asset-tracking
+index: 0
+meta_description: "The Ably asset tracking solution provides a way to track multiple assets in realtime."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Mapbox location enhancement, Ably realtime"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Key participants#key-participants
+    - Resolution policy#resolution-policy
+    - See also#see-also
+---
+
+The Ably asset tracking solution provides a way to track multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The asset tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
```

Comment:
> In the first sentence - probably makes sense to clarify "location of multiple assets".

Not sure where I say that! Also, asset is defined in the next sentence.

> should we use this naming here as well, instead of "Ably asset tracking solution"?

Yes we should! Fixed.

Reply:
Fixed. Clarified wording.